ads/_config.js

```diff
@@ -845,6 +845,10 @@ export const adConfig = {
     ],
   },
 
+  tracdelight: {
+    prefetch: 'https://scripts.tracdelight.com/amp.js',
```

Comment:
There is a `renderStart` API that helps to deliver better user experiences, and we highly recommend you implement it. Please refer to the doc [here](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis) and let me know if there's any questions.